RecoMuon/MuonIdentification/plugins/MuonIdProducer.cc

```diff
@@ -637,6 +636,10 @@ void MuonIdProducer::produce(edm::Event& iEvent, const edm::EventSetup& iSetup)
      }
    }
 
+   // muon arbitration
+   fillArbitrationInfo( outputMuons.get() );
+   arbitrateMuons( outputMuons.get(), caloMuons.get() );
```

Comment:
similar question (as for earlyMuons used in iter tk seeding) for earlyDisplacedMuons.
And, also for muonsFromCosmics, muonsFromCosmics1Leg which are running both in pp and in proper cosmic data taking.
At least for proper cosmics it may be better to keep unarbitrated parts.